### PR TITLE
adds tests for listInfiniteScrolling

### PR DESCRIPTION
**Pull request by codefriar**
List Infinite Scrolling lwc component.

On Behalf of Heather.

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: force-app/main/default/lwc/listInfiniteScrolling/__tests__/listInfiniteScrolling.test.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            );
+
+            expect(dataTableEl).not.toBeNull();
+            expect(dataTableEl.data).toEqual(
+                mockGetAccountsPaginatedRecords.records
+            );
+        });
+    });
+
+    it('does not display a data table when @wire returns error', () => {
```
Comment (pozil) [inline force-app/main/default/lwc/listInfiniteScrolling/__tests__/listInfiniteScrolling.test.js:39]: Test name doesn't reflect what the test does.
Can you split that into two tests: one that checks the error panel and one that ensures that table is not rendered when we have an error?

File: force-app/main/default/lwc/listInfiniteScrolling/__tests__/listInfiniteScrolling.test.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                expect(dataTableEl).not.toBeNull();
+                expect(dataTableEl.data).toEqual(
+                    mockGetAccountsPaginatedRecords.records
+                );
+                expect(
+                    getAccountsPaginatedAdapter.getLastConfig().pageToken
+                ).toBe(0);
+            })
+            .then(() => {
+                // Toggle checkbox to show details
```
Comment (pozil) [inline force-app/main/default/lwc/listInfiniteScrolling/__tests__/listInfiniteScrolling.test.js:80]: Fix comment

File: force-app/main/default/lwc/listInfiniteScrolling/__tests__/listInfiniteScrolling.test.js (hunk 1 above)
Comment (codefriar) [reply force-app/main/default/lwc/listInfiniteScrolling/__tests__/listInfiniteScrolling.test.js:39]: good call. incoming addition

**Comment by codefriar** [comment]
@pozil please reeval

**Pull request merged**